samples/tfm/tfm_cert/README.rst

```diff
@@ -0,0 +1,205 @@
+.. _tfm_cert:
+
+TF-M: PSA Certified Level 2
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample is a reference for how to achieve PSA Certified Level 2 using TF-M on nRF devices.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+This sample uses Trusted Firmware-M, nRF Secure Immutable bootloader and MCUboot to satisfy the Platform Security Architecture (PSA) Certified Level 2 requirements.
+The sample prints information about the identity of the device and the firmware versions that are currently running.
+
+Building and running
+********************
+
+The sample requires that the device is provisioned using the :ref:`provisioning image <provisioning_image>` sample.
```

Comment:
When are we counting on having merged [provisioning image sample ](https://github.com/nrfconnect/sdk-nrf/pull/9033)? Doc build for this PR will keep failing due to the link until the provisioning image sample is merged into main.

Reply:
Fingers crossed, very soon.